cluster/charts/rook/templates/deployment.yaml

```diff
@@ -65,6 +65,11 @@ spec:
 {{- end }}
         resources:
 {{ toYaml .Values.resources | indent 10 }}
+
+{{- if .Values.nodeSelector }}
+      nodeSelector:
+{{ toYaml .Values.nodeSelector | indent 8 }}
+{{- end }}
     {{- if .Values.rbacEnable }}
       serviceAccountName: rook-operator
     {{- end }}
```

Comment:
Could you remove the space before the `{{` for those two lines 73 and 75 in this PR too.